App/Game.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+using App.Players;
+using App.UI;
+
+namespace App
+{
+    public class Game
+    {
+        private Dictionary<string, Player> players;
+        private Board board;
+        private string turn;
+
+        public Game(Player playerOne, Player playerTwo, Board.Dimensions boardSize)
+        {
+            this.players = new Dictionary<string, Player>()
+            {
+                {Board.Marks.x.ToString(), playerOne},
+                {Board.Marks.o.ToString(), playerTwo}
+            };
+            this.board = new Board(boardSize);
+            this.turn = Board.Marks.x.ToString();
+        }
+
+        public virtual Board GetBoard()
+        {
+            return this.board;
+        }
+
+        public void PrintBoard()
+        {
+            MessageHandler.PrintBoard(board.GetGrid(), this.players);
+        }
+
+        public void Run()
+        {
+            while (!this.IsOver())
+            {
+                this.PlayTurn();
+                this.SwapTurn();
+            }
+            
+            this.PrintResults();
+        }
+
+        private void PlayTurn()
+        {
+            Player currentPlayer = this.players[this.turn];
+            int pos = currentPlayer.Move(this, this.turn);
+            this.board.SetField(pos, this.turn);
+        }
+
+        private bool IsOver()
+        {
+            return this.board.HasWinner() || this.board.IsTied();
+        }
+
+        private void SwapTurn()
+        {
+            // swap next whose turn it will be next
```

Comment:
Remove comments in code, I think the method name is descriptive enough to know what's happening.

Reply:
✅